site/content/docs/03-run-time.md

```diff
@@ -1081,3 +1082,28 @@ const { head, html, css } = App.render({
 	answer: 42
 });
 ```
+
+You can call the `.render()` with the following parameters:
+
+| option | default | description |
+| --- | --- | --- |
+| `props` | `{}` | An object of properties to supply to the component
+| `options` | `{}` | An object of options
+
+The `options` object takes in the following properties:
+| key | default | description |
+| --- | --- | --- |
+| `slots` | `{}` | slots to supply to the component
```

Comment:
I don't think we want to even mention this in the docs until we settle on an official API. I'd be in favor of also renaming it to `$$slots` to make it less discoverable and more obvious that it's a private API.

Reply:
Well will it be private API? Because on one of the use cases would be to use it to set up stuff during testing.

Reply:
Until we figure out what we want the slots API to look like, it should be private, yes. There's technically already a way to do this in the client side constructor as well, but we don't want to suggest anyone use that API either. This PR is for adding a public API for setting context in the constructor, and we shouldn't be bringing along other unrelated API commitments with that.